test/apiserver/makefile

```diff
@@ -40,11 +43,21 @@ config: config/konk.yaml config/infra.yaml
 deploy-%: config/%.yaml
 	kubectl apply -f $<
 
-kind-load: KIND=false
-kind-load: QUAY_IMG=quay.io/coreos/etcd:v3.4.13
-kind-load: image
-	docker pull ${QUAY_IMG}
-	$(KIND) load docker-image ${QUAY_IMG} --name ${KIND_NAME}
+.kind-load-etcd-${ETCD_TAG}:
+	docker pull ${ETCD_IMG}
+	$(KIND) load docker-image ${ETCD_IMG} --name ${KIND_NAME}
```

Comment:
Why not just allow kind to pull the image from quay.io?

Reply:
done